doc/test_organization/managing_tests_dependencies.qbk

```diff
@@ -7,17 +7,31 @@
 
 [section:tests_dependencies Managing test dependencies]
 
-Dependencies among tests are managed by means of [link boost_test.tests_organization.decorators decorators]. 
+In general, you should make sure that test cases are independent on one another.
+For any two test cases `TA` and `TB`, `TB` must not take for granted that `TA` has already executed,
+even if `TA` is declared before `TB` in the same translation unit.
+
+The only ordering-related guarantee that __UTF__ makes by default is that if test cases `TA` and `TB`
+are declared in the same test suite, no test case (call it `TX`) from any other test suite is executed between `TA` and `TB`,
+even if the declaration of `TX` appears between the declarations of `TA` and `TB`.
+In other words, all tests from a suite are executed in one go, even if the test suite namespace is opened multiple times.
```

Comment:
To me, these are orthogonal:
- you make a test entry/exit to ensure that every test under the test suite have executed this fixture once (which is different from have executed this fixture "always")
- the fact that we execute all test units under a test suite in one go can be considered as a minimal number of execution for this suite entry/exit fixtures, eg if the fixture one is costly. This is somehow not the case when shuffling (I guess).